_api-reference/ingest-apis/processors-reference/append.md

```diff
@@ -0,0 +1,57 @@
+---
+layout: default
+title: Append
+parent: Ingest processors 
+grand_parent: Ingest APIs
+nav_order: 10
+---
+
+# Append
+
+The `append` proccessor is used to add additional fields or values to a document. The syntax for the `append` processor is: 
+
+```json
+{
+    "append": {
+        "field": "field_name",
+        "value": ["value1"]
+    }
+}
+```
+
+## Configuration parameters
+
+The following table lists the required and optional parameters for the `append` processor.
+
+**Parameter** | **Required** | **Description** |
+|-----------|-----------|-----------|
+`field`  | Required  | Name of the field where the data should be appended. |
+`value`  | Required  | Value to be appended. This can be a static value, a dynamic value derived from existing fields, or a value obtained from external lookups.  |
+`ignore_empty_fields`  | Optional  | If set to true, empty values will be ignored when appending then to the target field.  | 
+`fail_on_error`  | Optional  | If set to true, the processor will fail it an error occurs. The default value is false.  |
+`allow_duplicates`  | Optional  | If set to false, the processor will not append values that already exist in the target field. The default value is set to true.  |
+`ignore_missing`  | Optional  | If set to true, the processor will ignore events that lack the target field. The default value is false.  |
+`description`  | Optional  | Brief description of the processor.  |  
+
+Following is an example of adding the `append` processor to an ingest pipeline.
```

Comment:
@kolchfa-aws Should we include an example of testing the processor in a document?

Reply:
I don't think we should include an example of testing the processor because all of them are tested in the same way. But we should definitely test the request as part of writing these docs, which you're already doing 🥇

Reply:
On a second thought, if a processor does something more complicated that is easier to see on an example, we can include a testing example to see the output.

Reply:
@kolchfa-aws Are you available to give these a doc review by 7/7?

Reply:
Will do